app/src/main/res/values/strings.xml

```diff
@@ -11,7 +11,7 @@
     <string name="scan_qr_code">Scan QR COde</string>
     <string name="email">Email</string>
     <string name="password">Password</string>
-    <string name="login">Login</string>
+    <string name="login">LOG IN</string>
```

Comment:
Don't capitalize strings. It is automatically done by buttons, and secondly, you can put textAllCaps if needed